### DISCUSSION TITLE
Multiplayer History Beatmaps and SR Don't Match osu!web

### DISCUSSION THREAD

**Discussion by zachmanthethird** [ppy/osu#16728] [Q&A] [answered]
![osu_2022-01-31_22-28-15](https://user-images.githubusercontent.com/74262407/151922487-37c782ce-5c34-47f2-8690-e07515d79169.png)
![image](https://user-images.githubusercontent.com/74262407/151922547-d1aee753-c7b8-4194-a926-b9118623b85b.png)
Shouldn't lazer be pulling from the same data source as osu!web? It clearly shows that there were no beatmaps played in lazer while osu!web shows the correct statistics and star rating.

**Comment by peppy** [comment]
Could you please explain which part is different for clarity?

**Reply by zachmanthethird** [reply]
osu!lazer reports all of these lobbies as never played any beatmaps and with 0*
Otherwise, yes the screens show the same information.

**Reply by Theighlin** [reply]
#14787

**Reply by zachmanthethird** [reply]
There it is; I guess I didn't use enough search terms...

**Comment by zachmanthethird** [comment, marked as answer]
Duplicate of #14787
